internal/k8s/client.go

```diff
@@ -34,14 +38,28 @@ func (kc k8sClient) InClusterConfig() (*rest.Config, error) {
 
 // NewKubeClientLocal creates a new kube client for local mode
 func NewKubeClientLocal(configPath string) (*kubernetes.Clientset, error) {
-	if _, err := os.Stat(configPath); err != nil {
-		return nil, ErrNoReadableKubeConfig
-	}
-	config, err := clientcmd.BuildConfigFromFlags("", configPath)
-	if err != nil {
-		return nil, err
+	var kubeconfig *rest.Config
+	if configPath == "" {
+		var err error
```

Comment:
Because `kubeconfig` is declared outside the scope. With `:=` I think it would initialize `kubeconfig` to the local scope rather than assigning to the outer one. I'll clean this up a bit